src/python/pants/backend/python/target_types.py

```diff
@@ -42,8 +42,9 @@ class PythonInterpreterCompatibility(StringOrStringSequenceField):
     """A string for Python interpreter constraints on this target.
 
     This should be written in Requirement-style format, e.g. `CPython==2.7.*` or `CPython>=3.6,<4`.
-
     As a shortcut, you can leave off `CPython`, e.g. `>=2.7` will be expanded to `CPython>=2.7`.
+
+    If this is left off, this will default to `--python-setup-interpreter-constraints`.
```

Comment:
I'm starting to think that we should reference options in documentation using the config file form, as this is what people will use in practice almost all the time. I.e., 

This will default to the value of `interpreter_constraints` in the `[python_setup]` scope.

What do you think?

Reply:
So long as we still say `[python_setup]`, I think that this is a great idea.